dev/merge_arrow_pr.py

```diff
@@ -421,8 +421,19 @@ def merge_pr(self, number, commit_title, commit_message):
         if response.status_code != 200 and 'merged' not in result:
             result['merged'] = False
             result['message'] += f': {url}'
+        else:
+            self.clear_pr_state_labels(number)
         return result
 
+    def clear_pr_state_labels(self, number):
+        url = f'{self.github_api}/issues/{number}/labels'
+        response = requests.get(url, headers=self.headers)
+        labels = response.json()
+        for label in labels:
+            if label['name'].startswith('awaiting'):
```

Comment:
Do all workflow labels start with "awaiting"? In any case, add a comment explaining this?

Reply:
yes, all workflow labels start with the `awaiting` prefix. I will add a comment